internal/ai/conversation.go

```diff
@@ -60,20 +63,32 @@ func ResumeConversation(
 		tools:        tools,
 		Messages:     history.Messages,
 
-		maxTokens: maxTokens,
+		maxOutputTokens: maxOutputTokens,
+		tokenLimit:      100000, // 100k token limit
 	}
 	return c, nil
 }
 
-// sendMessage is the internal implementation with a boolean parameter to specify caching
+// SendMessage sends a message with caching enabled
```

Comment:
SendMessage is an exported function, and caching is an implementation detail. Don't mention caching in the comment